docs/features/models/kbqa.rst

```diff
@@ -4,54 +4,94 @@ Knowledge Base Question Answering (KBQA)
+
+* Substring extracted by the NER model is used for entity linking. Entity linking preforms matching the substring with one of the Wikidata entities. Matching is based on Levenshtein distance between the substring and an entity description. The result of the matching procedure is a set of candidate entities. The reset is search of the entity among this set with one of the top-k relations predicted by classification model.
+
+Model for simple question answering with syntactic parser uses UDPipe for parsing of syntactic tree to extract candidate entity and relation substrings.
 
 Use the model
 -------------
 
-At the moment the model that solves KBQA task is available only for Russian language.
 Any pre-trained model can be used for inference from both Command Line Interface (CLI) and Python. Before using the
 model make sure that all required packages are installed using the command:
 
 .. code:: bash
 
+    python -m deeppavlov install kbqa_cq
+    python -m deeppavlov install kbqa_cq_online
     python -m deeppavlov install kbqa_rus
+    python -m deeppavlov install kbqa_tree
 
 To use a pre-trained model from CLI use the following command:
 
 .. code:: bash
 
+    python deeppavlov/deep.py interact kbqa_сq [-d]
+    python deeppavlov/deep.py interact kbqa_cq_online [-d]
     python deeppavlov/deep.py interact kbqa_rus [-d]
+    python deeppavlov/deep.py interact kbqa_tree [-d]
 
-where ``kbqa_rus`` is the name of the config and ``-d`` is an optional download key. The key ``-d`` is used
+where ``kbqa_cq`` and others are the names of configs and ``-d`` is an optional download key. The key ``-d`` is used
 to download the pre-trained model along with embeddings and all other files needed to run the model. Also command
 ``download`` is possible,
 
 
-Available config:
 
-.. table::
-    :widths: auto
+KBQA model for complex question answering can be used from Python using the following code:
 
-    +-----------------------------------------------+-------------------+-----------------+------------+------------+
-    | Model                                         | Dataset           | Embeddings Size | Model Size |  Wikidata  |
-    +===============================================+===================+=================+============+============+
-    | :config:`kbqa_rus <kbqa/kbqa_rus.json>`       | Simple Questions  |     7.7 GB      |   8.9 MB   |   1.1 GB   |
-    |                                               | and Zero-Shot IE  |                 |            |            |
-    +-----------------------------------------------+-------------------+-----------------+------------+------------+
+.. code:: python
+
+    from deeppavlov import configs, build_model
 
+    kbqa_model = build_model(configs.kbqa.kbqa_cq, download=True)
+    kbqa_model(['When did Jean-Paul Sartre move to Le Havre?'])
+    >>> ["1931-01-01"]
+    kbqa_model(['How many sponsors are for Juventus F.C.?'])
+    >>> [4]
```

Comment:
Doesn't work - model returns "Not found"

Reply:
I sent new checkpoints of RelRanker to share. Some questions cannot be answered due to using not full version of wikidata. I sent to share not full version because full version is rather big (takes up 73 Gb), it can be downloaded from http://www.rdfhdt.org/datasets/